pay-api/src/pay_api/resources/__init__.py

```diff
@@ -78,3 +76,5 @@
 API.add_namespace(INVOICE_RECEIPT_API, path='/payments/<int:payment_id>')
 
 API.add_namespace(BCOL_API, path='/bcol/accounts/<string:account_id>/users/<string:user_id>')
+
+API.add_namespace(SERVICE_STATUS_API, path='/status/<string:service_name>')
```

Comment:
May be we can design this rest service to handle below;
1) /status : which will return all the dependant system status
2) /status/:system : which is the current implementation to return specific service status.
Thoughts?

Reply:
Yes, we can think about to support return all services endpoint.